include/fastrtps/rtps/writer/StatefulWriter.h

```diff
@@ -192,6 +192,7 @@ class StatefulWriter : public RTPSWriter
      */
     inline size_t getMatchedReadersSize() const
     {
+		std::lock_guard<RecursiveTimedMutex> guard(mp_mutex);
```

Comment:
This method does not seem to be used anywhere, should we remove it, or should we keep it in case users are using it? If we keep it, fix indentation please.